src/Descriptors/Schema/Schema.php

```diff
@@ -111,7 +111,7 @@ public function store(Route $route): OASchema
             OASchema::object('attributes')
               ->properties(...$fields->get('attributes')),
             OASchema::object('relationships')
-              ->properties(...$fields->get('relationships'))
+              ->properties(...$fields->get('relationships') ?: [])
```

Comment:
Is this an unrelated bugfix?

Reply:
This is kind of an unrelated bugfix? I walked into this issue when trying to test the descriptions in a working application where a schema had no relations. So I decided to fix it while working on this feature.

Reply:
Yeah, i understand you ran into this, but probably because of a generic problem not related to this specific PR. 

In this case its fine to add this in the PR since we are in the middle of dev. But normally i'd probably ask for a seperate issue for that :)

Reply:
Understandable. I will remember that for the next time!